cmd/snap/cmd_info.go

```diff
@@ -76,6 +76,40 @@ func init() {
 		}), nil)
 }
 
+func (iw *infoWriter) maybePrintHealth() {
+	if iw.localSnap == nil {
+		return
+	}
+	health := iw.localSnap.Health
+	if health == nil {
+		if !iw.verbose {
+			return
+		}
+		health = &client.SnapHealth{
+			Status:  "unknown",
+			Message: "health has not been set",
+			Code:    "snapd-no-health",
```

Comment:
should be "snapd-no-health-set", for similarity with the "hook" one

Reply:
I thought a bit more, this is the ground case for "unknown", no?  it probably can be without code. Am I missing something?

Reply:
this is indeed the default case, any snap that has no hook and has never been set-health will fall into this case.

Reply:
adjusted to not have a code